src/chttpd_cors.erl

```diff
@@ -360,7 +399,7 @@ get_origin(Req) ->
         undefined ->
             undefined;
         Origin ->
-            ?l2b(to_lower(Origin))
+            ?l2b(Origin)
```

Comment:
Actually CORS specs [point](https://www.w3.org/TR/cors/#origin-request-header) to "The Web Origin Concept" RFC and hence require host to be ["converted to lowercase using the ascii-casemap collation"](https://tools.ietf.org/html/rfc6454#section-4) (subsection 5). So I guess we still need to_lower here